Add and cover upgrade to nginx IC App v1.6.7 (#220)
giantswarm/releases: release-notes/azure/v11.2.2.md

This release contains improvements in the upgrade procedure.
We have improved new nodes discovery at the stage where the cluster size is doubled with new versions of the machines.

> This will make sure that in case of upgrade disruptions, the process will follow its original path.

Reviewer: ```suggestion
This will make sure that in case of upgrade disruptions the process will recover and continue as planned.
```
@T-Kukawka
